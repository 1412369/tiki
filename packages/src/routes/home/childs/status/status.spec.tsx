import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Status from './status';
const meta = {
  total: 36,
  types: ['standard', 'vip', 'deluxe'],
  standard: { price: 10000 },
  vip: { price: 12000 },
  deluxe: { price: 15000 },
};
describe('snapshot Status', function() {
  it('Should render Status without crashing', function() {
    const wrapper = renderer.create(<Status meta={meta} />);
    expect(wrapper).toMatchSnapshot();
  });
});
