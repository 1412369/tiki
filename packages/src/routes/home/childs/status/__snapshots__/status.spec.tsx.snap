// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot Status Should render Status without crashing 1`] = `
<div
  className="sc-bdVaJa iKTrfL"
>
  <ul>
    <li
      style={
        Object {
          "color": "#464646",
        }
      }
    >
      standard
      : 
      ₫ 10,000
    </li>
    <li
      style={
        Object {
          "color": "#008e00",
        }
      }
    >
      vip
      : 
      ₫ 12,000
    </li>
    <li
      style={
        Object {
          "color": "#f76f6f",
        }
      }
    >
      deluxe
      : 
      ₫ 15,000
    </li>
  </ul>
</div>
`;
